amr-wind/wind_energy/ABLFieldInit.cpp

```diff
@@ -33,6 +34,9 @@ ABLFieldInit::ABLFieldInit()
 
     pp_abl.query("init_tke", m_tke_init);
 
+    // Use input from netcdf file
+    pp_abl.query("initial_condition_input_file", m_ic_input);
+
```

Comment:
We probly need to perform a few checks. First, that there are no mesh refinement zones enabled. Second, this option will fail or default to some baseline if netcdf is not enabled.

Reply:
these fixes have been added.